### PR TITLE
refactor: translation upgrade to angular-translate

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
MFRWW-719

### Requirements

* Upgrade from i18n translator to angular-translate

### Description of the Change

The upgrade to angular-translate has been done.

**Link**
https://github.com/ovh-ux/ovh-manager-web/pull/534

### CODE REVIEW

File: src/microsoft/office/license/user/order/microsoft-office-license-user-order.html [hunk 1]
```diff
@@ -45,7 +45,6 @@
         </div>
 
         <div data-wizard-step>
-            <p data-translate="microsoft_office_license_order_user_confirm"></p>
```
Comment (FredericEspiau) [inline src/microsoft/office/license/user/order/microsoft-office-license-user-order.html]: Why did you deleted this line entirely ?

**Comment by ovh-cds** [comment]
CDS Report build#299.0 ✘  
* build
  * build ✘  
  * test ✘

File: src/microsoft/office/license/user/order/microsoft-office-license-user-order.html (hunk 1 above)
Comment (ganeshkumar1989) [reply src/microsoft/office/license/user/order/microsoft-office-license-user-order.html]: The corresponding translation is not present. In old translator, this was just inserting a blank paragraph.

File: src/microsoft/microsoft.app.js [hunk 2]
```diff
@@ -28,11 +28,8 @@ angular.module('Module.microsoft', [
...
-        return translator.load(['microsoft']).then(() => translator);
-      }],
     },
+    translations: ['microsoft'],
   });
-}]).run(['translator', 'MicrosoftOfficeLicenseService', function (translator) {
-  translator.load(['microsoft']);
-}]).constant('MICROSOFT_GUIDE_URLS', {});
+}])
+.constant('MICROSOFT_GUIDE_URLS', {});
```
Comment (JeremyDec) [inline src/microsoft/microsoft.app.js]: `error  Expected indentation of 2 spaces but found 0  indent`

File: src/microsoft/office/license/user/order/microsoft-office-license-user-order.html (hunk 1 above)
Comment (ganeshkumar1989) [reply src/microsoft/office/license/user/order/microsoft-office-license-user-order.html]: 👍

File: src/microsoft/microsoft.app.js (hunk 2 above)
Comment (ganeshkumar1989) [reply src/microsoft/microsoft.app.js]: This has been fixed and the build is now successful

Comment (JeremyDec) [reply src/microsoft/microsoft.app.js]: Ok good :D

**Pull request merged**